app/templates/scheduled-job/details.hbs

```diff
@@ -0,0 +1,177 @@
+{{page-title "Scheduled Job Details"}}
+{{breadcrumb "Details" route="scheduled-job.details"}}
+
+<AuMainContainer as |m|>
+  <m.sidebar>
+    <div class="au-c-sidebar">
+      <div class="au-c-sidebar__content">
+        <div class="au-u-padding au-u-padding-top">
+          <AuLink
+            @route="overview.scheduled-jobs"
+            @skin="button"
+            @icon="nav-left"
+            @iconAlignment="left"
+            @width="inline">
+            Back to overview
+          </AuLink>
+          {{!--Job info--}}
+          <AuHeading
+            @level="3"
+            @skin="3"
+            class="au-u-margin-top au-u-margin-bottom">
+            Job details
+          </AuHeading>
+          <div class="au-u-flex--inline au-u-flex--wrap au-u-flex--spaced">
+            <div>
+              <AuLabel>Title</AuLabel>
+              <p class="au-u-word-break">
+                {{this.job.title}}
+              </p>
```

Comment:
Hmm, in my opinion it is not used in the same way. In the above example, we are not describing what "title" means, but what the value is for the title property of the job object. I'm all for using the semantic elements, but they have to be the correct ones to use.

Now I'm nitpicking on your nitpick :smile:

Reply:
In HTML5 the `<dl>` element was extended so that it can also be used for key-value metadata: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#metadata

So, AFAIK it's a valid use-case.

> I'm all for using the semantic elements, but they have to be the correct ones to use.

Agreed. So using a `<label>` for something else than an input shouldn't be done either 😄.

Reply:
You're right again. I'll change it in the four places that is done.